frontend/src/components/GamePhaseSpecificSidebar.tsx

```diff
@@ -22,14 +22,18 @@ const GamePhaseSpecificSidebar: React.FunctionComponent<Props> = ({
     case "BATTLE":
       const battleGameState = gameState as BattleGameState;
```

Comment:
Wait, this line is not needed... typescript knows that type is BATTLE and thus the type is BattleGameState. Removing this line should allow you to remove the `?` from the next lines (`battleGameState.battleStatistics`)

Reply:
Hmm. I'll check it. But Maciek wrote it some time ago, and I don't remember why, but it didn't seem to work without it. Furthermore, the '?' are there, because right now, there is a case that battleStatistics might be undefined (when someone disconnects in the middle of a game). But I'll test it.